features/admin/manage_users.feature

```diff
@@ -0,0 +1,27 @@
+@manage-users
+Feature: Admin manager can manage users
+
+Scenario: Admin Manager user can log in and suspend admin users
+  Given I am logged in as the production admin-manager user
+  And I am on the /admin page
+  And I click the 'Manage users' link
+  #TODO change this email address
+  And I click the summary table Edit link for '14288@user.marketplace.team'
+  When I choose the 'Suspended' radio button
+  And I click the 'Update user' button
+  Then I see a success banner message containing '14288@user.marketplace.team has been updated.'
+  And I click the summary table Edit link for '14288@user.marketplace.team'
+  And I choose the 'Active' radio button
+  And I click the 'Update user' button
+
+Scenario Outline: Correct users cannot access the manage users page
```

Comment:
This is kind of an odd one.  This behaviour is 100% covered by unit tests, but maybe it's good to have it tested in integration with the API too.

Reply:
I quite like it in terms of tests as documentation. It's really clear who should have what flow and what should happen if they don't. I'm going to keep it unless you feel strongly. I'm hoping it will make people more aware of the different roles when changing things too.

Reply:
No I don't feel strongly at all - happy to keep it in.